clients/lsp-javascript.el

```diff
@@ -44,7 +44,7 @@
 
 (defun lsp-typescript-javascript-tsx-jsx-activate-p (filename &optional _)
   "Check if the javascript-typescript language server should be enabled based on FILENAME."
-  (or (string-match-p (rx (one-or-more anything) "." (or "ts" "js") (opt "x") string-end) filename)
+  (or (string-match-p (rx (one-or-more anything) "." (or "mjs" (or "ts" "js") (opt "x")) string-end) filename)
```

Comment:
Actually, I think this causes a regression: `tsx` wouldn#t be matched anymore, because the `opt` is now part of another `or` branch:
```suggestion
  (or (string-match-p (rx (one-or-more anything) "." (or "mjs" (: (or "ts" "js") (opt "x"))) string-end) filename)
```

Reply:
Is `mjsx` a thing?

Reply:
mjsx is not a thing

Reply:
Thanks for clarifying; then my `suggestion` should be correct.